File: javascript-closure-angular-client/API/Client/EntityLocation.js
goog.provide('API.Client.EntityLocation');

/**
 * @record
 */
API.Client.EntityLocation = function() {}

/**
 * Street Name
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.street;

/**
 * Neighborhood Name
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.neighborhood;

/**
 * Brazilian Zip Code
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.zipcode;

/**
 * City Code (IBGE)
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.cityCode;

/**
 * City Name
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.cityName;

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.EntityLocation.prototype.state;

/**
 * Country Code
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.countryCode;

/**
 * Use ISO 3166-1 alpha-3 codes
 * @type {!string}
 * @export
 */
API.Client.EntityLocation.prototype.country;

File: async-scala-client/src/main/scala/io/swagger/client/model/EntityLocation.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class EntityLocation (
  street: Option[String],  // Street Name
neighborhood: Option[String],  // Neighborhood Name
zipcode: String,  // Brazilian Zip Code
cityCode: Option[String],  // City Code (IBGE)
cityName: Option[String],  // City Name
state: Option[StateEnum],
countryCode: Option[String],  // Country Code
country: Option[String]  // Use ISO 3166-1 alpha-3 codes
)
